x-pack/plugins/infra/server/lib/adapters/log_entries/kibana_log_entries_adapter.ts

```diff
@@ -152,6 +160,35 @@ export class InfraKibanaLogEntriesAdapter implements LogEntriesAdapter {
       : [];
   }
 
+  public async getLogItem(
+    request: InfraFrameworkRequest,
+    id: string,
+    sourceConfiguration: InfraSourceConfiguration
+  ) {
+    const search = (searchOptions: object) =>
+      this.framework.callWithRequest<LogItemHit, {}>(request, 'search', searchOptions);
+
+    const params = {
+      index: sourceConfiguration.logAlias,
+      terminate_after: 1,
+      body: {
+        size: 1,
+        query: {
+          match: {
```

Comment:
There is an [`ids` query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-ids-query.html), which might be more robust to potential future changes in Elasticsearch's treatment of the `_id` field.